GRAPHICS/GetScaleformMovieFunctionReturnInt.md

```diff
@@ -1,12 +1,12 @@
 ---
 ns: GRAPHICS
-aliases: ["0x2DE7EFA66B906036"]
+aliases: ["0x2DE7EFA66B906036", "_GET_SCALEFORM_MOVIE_FUNCTION_RETURN_INT"]
 ---
-## _GET_SCALEFORM_MOVIE_FUNCTION_RETURN_INT
+## GET_SCALEFORM_MOVIE_FUNCTION_RETURN_VALUE_INT
 
 ```c
 // 0x2DE7EFA66B906036 0x2CFB0E6D
```

Comment:
Joaat hash does not match the new name. Please prefix the native name with `_` in that case.

(`GET_SCALEFORM_MOVIE_FUNCTION_RETURN_VALUE_INT` = `0xB86A32AE`, this native's Joaat hash is `0x2CFB0E6D`)

Also the file name should be changed when changing the native name.